shotizam.go

```diff
@@ -31,6 +31,7 @@ var (
 	base    = flag.String("base", "", "base file to diff from; must be in json format")
 	mode    = flag.String("mode", "sql", "output mode; tsv, json, sql, nameinfo")
 	sqlite  = flag.Bool("sqlite", false, "launch SQLite on data (when true, mode flag is ignored)")
+	sqldb   = flag.String("sqldb", "", "save SQLite db (when set, mode and sqlite flag is ignored)")
```

Comment:
as written, this sounds like a boolean flag. specify that this is the target filename to write the DB to?

Reply:
@bradfitz yep, it's the target db filename.

how about `-o filename`, then we can use it like this:

```
shotizam --sqlite   -o bin-size.db  /some/go.binary
shotizam --mode csv -o bin-size.csv /some/go.binary
shotizam --mode sql -o bin-size.sql /some/go.binary
```

Reply:
Works for me!

Reply:
working on it..

Reply:
it's ready.

btw `--mode nameinfo` also support `-o`

```
shotizam --sqlite -o bin-size.db   SELF
shotizam --mode nameinfo -o bin-size.txt SELF
```